libs/live-fns/pypi.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('../axios.js')
+
+module.exports = async function (method, ...args) {
+  const endpoint = `https://pypi.org/pypi/${args[0]}/json`
+  const { info } = await axios.get(endpoint).then(res => res.data)
+
+  switch (method) {
+    case 'v':
+      return {
+        subject: 'pypi.org',
+        status: 'v' + info.version,
+        color: info.version[0] === '0' ? 'orange' : 'blue'
+      }
+    default:
+      return {
+        subject: 'pypi',
```

Comment:
switch it to `pypi.org` too? for consistency? Or just make them both only `pypi`.

Reply:
+1 for both `pypi`

Like pip already did in pypi.org
https://pypi.org/project/pip/

Reply:
switched all to `pypi`